### PR TITLE
stm32cube: update stm32XX cube version

### PR DESCRIPTION
**Pull request by FRASTM**
Update Cube version for following series
(from https://github.com/STMicroelectronics/STM32Cube)

- stm32cube: update stm32wb to version V1.3.0  (commit 374333fb168689f936e8e99734bf3d2a84322a9f) 
- stm32cube: update stm32g0 to version V1.3.0 (commit e209cea2bbbd5fe381d032b49d19261d2c1fc4d1)
- stm32cube: update stm32f0 to version V1.11.0 (commit 08833e5289d1c12be6d804a1e327268db52350f2)
- stm32cube: update stm32f3 to version V1.11.0  (commit 2a412a476a300808806bc1a4c14ba71f2a6717b2)

Based on #30, hence also includes

- stm32cube: update stm32h7xx cube version to V1.5.0 (commit a08d1dcc18f1d93083dcd62083973016ac847320)
- stm32cube: update stm32f2xx cube version to V1.8.0 (commit b4226d57c8bb0207e76ddb2402eec04c6ef8e037)
- stm32cube: update stm32f1xx cube version to V1.8.0 (commit a515c6743bd140d21cd8173ca26d09207f318632)

Signed-off-by: Francois Ramu francois.ramu@st.com

### CODE REVIEW

**Comment by FRASTM** [comment]
This PR requires the script [https://github.com/zephyrproject-rtos/hal_stm32/pull/26](url)

**Comment by erwango** [comment]
Please rebase on https://github.com/zephyrproject-rtos/hal_stm32/pull/30 so all modules are delivered in one common PR.

File: stm32cube/stm32f3xx/README [hunk 1]
```diff
@@ -35,6 +35,7 @@ License Link:
    https://opensource.org/licenses/BSD-3-Clause
 
 Patch List:
+--> please check that the following list is still valid:
```
Comment (erwango) [inline stm32cube/stm32f3xx/README]: Please remove this  line if check has been done

**Comment by FRASTM** [comment]
with update in the README

File: stm32cube/stm32mp1xx/README [hunk 2]
```diff
@@ -35,6 +35,7 @@ License Link:
    https://opensource.org/licenses/BSD-3-Clause
 
 Patch List:
+--> please check that the following list is still valid:
```
Comment (erwango) [inline stm32cube/stm32mp1xx/README]: Is the patch list valid ?

File: stm32cube/stm32mp1xx/README [hunk 3]
```diff
@@ -35,62 +35,15 @@ License Link:
    https://opensource.org/licenses/BSD-3-Clause
 
 Patch List:
---> please check that the following list is still valid:
 
-    * Initial version 1.0.0
-
-    * Add stm32mp1 exti ll driver
-    The stm32mp1 exti ll driver is mandatory for compilation.
```
Comment (erwango) [inline stm32cube/stm32mp1xx/README]: Please move to mp1 update commit

**Comment by FRASTM** [comment]
with update in the README

**Comment by galak** [comment]
@erwango should this still be marked DNM?

**Comment by erwango** [comment]
@galak, we miss HW testing, we'll try to complete this week.

**Comment by erwango** [comment]
@galak ready to merge!

**Pull request merged**